src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -2471,7 +2471,7 @@ End Class
 "<Extension> Function IEnumerable(Of 'a).ToArray() As 'a()
 
 Anonymous Types:
-    'a is New With { .x As Integer }")
+    'a is New With { Key .x As Integer }")
```

Comment:
Note: this is not the correct behavior. But it's because we haven't hooked up the SymbolKey API to properly pass along the right data.  That happens in https://github.com/dotnet/roslyn/pull/14307.  In that PR this change will be undone.